i18n/it.json

```diff
@@ -1940,7 +1940,7 @@
   },
   {
     "id": "api.team.add_members.user_denied",
-    "translation": "Questa squadra è gestita dai grupi.  Questo utente non appartiene a un gruppo sincronizzato con questa squadra."
+    "translation": "Questa squadra è gestita dai gruppi.  Questo utente non appartiene a un gruppo sincronizzato con questa squadra."
```

Comment:
```suggestion
    "translation": "Questa squadra è gestita dai gruppi. Questo utente non appartiene a un gruppo sincronizzato con questa squadra."
```

Reply:
@isacikgoz I believe changes like this need to be done directly in the translation server.

Reply:
@amyblais submitted to the translation server. 👍